server/utils/helpers.js

```diff
@@ -0,0 +1,7 @@
+const buildResponse = (isSuccess, message, data = {}) => {
+  if (isSuccess) {
+    return { isSuccess, message, data };
```

Comment:
why are you sending isSuccess on return? are you checking somewhere else?

Reply:
We want to have a common response format for all the API's. So we decided to have response is form like { isSuccess : true/false, message : '' ", data : [] / {} }. Whennever isSuccess is false, there will be a custom message which can be displayed on UI.

Reply:
@praveshtora can we not check http response codes 200s/400s/500s instead of another variable